Source/Core/DolphinWX/Cheats/CheatsWindow.cpp

```diff
@@ -154,7 +153,7 @@ void wxCheatsWindow::OnEvent_Close(wxCloseEvent& ev)
 void wxCheatsWindow::UpdateGUI()
 {
 	// load code
-	SCoreStartupParameter parameters = SConfig::GetInstance().m_LocalCoreStartupParameter;
+	SConfig& parameters = SConfig::GetInstance();
```

Comment:
`UpdateGUI` only calls const methods on `parameters`, so making this a reference doesn't lead to any difference in behavior.